### PR TITLE
chore(csi): add csi-operators 1.10.0 yamls

### PR DESCRIPTION
**Pull request by prateekpandey14**
Signed-off-by: prateekpandey14 <prateek.pandey@mayadata.io>

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/openebs]`)

### CODE REVIEW

File: docs/csi-operator-1.10.0-ubuntu-18.04.yaml [hunk 1]
```diff
@@ -0,0 +1,993 @@
...
+            - "--v=5"
+            - "--driver-requires-attachment=true"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+        - name: openebs-csi-plugin
```
Comment (mittachaitu) [inline docs/csi-operator-1.10.0-ubuntu-18.04.yaml:730]: Can we change this to `openebs-cstor-csi-plugin`? Or leaving as is fine?

File: docs/csi-operator-1.10.0-ubuntu-18.04.yaml [hunk 2]
```diff
@@ -0,0 +1,993 @@
...
+          image: quay.io/openebs/cstor-csi-driver:1.10.0
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: OPENEBS_CONTROLLER_DRIVER
+              value: controller
+            - name: OPENEBS_CSI_ENDPOINT
+              value: unix:///var/lib/csi/sockets/pluginproxy/csi.sock
+            - name: OPENEBS_CSI_API_URL
+              value: https://api.openebs.com/
+            - name: OPENEBS_MAPI_SVC
```
Comment (mittachaitu) [inline docs/csi-operator-1.10.0-ubuntu-18.04.yaml:740]: Do we still need it?

File: docs/csi-operator-1.10.0.yaml [hunk 3]
```diff
@@ -0,0 +1,982 @@
...
+            - name: OPENEBS_NODE_DRIVER
+              value: node
+            - name: OPENEBS_API_URL
+              value: https://api.openebs.com/
+            - name: OPENEBS_MAPI_SVC
+              value: maya-apiserver-service
+            - name: OPENEBS_NAMESPACE
+              value: openebs
+            - name: REMOUNT
+              value: "true"
```
Comment (mittachaitu) [inline docs/csi-operator-1.10.0.yaml:949]: We are enabling REMOUNT by default. Should we need to mention in the release doc.

File: docs/csi-operator-1.10.0.yaml [hunk 4]
```diff
@@ -0,0 +1,982 @@
+# This manifest deploys the OpenEBS CSI control plane components,
+# with associated CRs & RBAC rules. This manifest has been verified
+# only with Ubuntu 16.04 and CentOS hosts, due to dependencies on
```
Comment (mittachaitu) [inline docs/csi-operator-1.10.0.yaml:3]: In CentOS-7.7 version, there is some problem with iscsi module right? Should we need to specify the supported versions?

Comment (prateekpandey14) [reply docs/csi-operator-1.10.0.yaml:3]: its supported, about the modules are not loading properly those steps we have in our docs

File: docs/csi-operator-1.10.0.yaml (hunk 3 above)
Comment (prateekpandey14) [reply docs/csi-operator-1.10.0.yaml]: Yes ,  i have disabled it, as we don't have much test cases yet and its alpha feature

File: docs/csi-operator-1.10.0-ubuntu-18.04.yaml (hunk 1 above)
Comment (prateekpandey14) [reply docs/csi-operator-1.10.0-ubuntu-18.04.yaml:730]: thats fine as we are having similar container name in director install as well

File: docs/csi-operator-1.10.0-ubuntu-18.04.yaml (hunk 2 above)
Comment (prateekpandey14) [reply docs/csi-operator-1.10.0-ubuntu-18.04.yaml:740]: let be there for now, will see later.

Comment (prateekpandey14) [reply docs/csi-operator-1.10.0-ubuntu-18.04.yaml:740]: removed

**Comment by prateekpandey14** [comment]
@mittachaitu  PR has been raised here https://github.com/openebs/charts/pull/99

**Pull request merged**